packages/components/src/unit-control/utils.ts

```diff
@@ -102,6 +102,126 @@ const allUnits: Record< string, WPUnitControlUnit > = {
 		a11yLabel: __( 'Points (pt)' ),
 		step: 1,
 	},
+	svw: {
+		value: 'svw',
+		label: isWeb ? 'svw' : __( 'Small viewport width (svw)' ),
+		a11yLabel: __( 'Small viewport width (svw)' ),
+		step: 0.1,
+	},
+	svh: {
+		value: 'svh',
+		label: isWeb ? 'svh' : __( 'Small viewport height (svh)' ),
+		a11yLabel: __( 'Small viewport height (svh)' ),
+		step: 0.1,
+	},
+	svi: {
+		value: 'svi',
+		label: isWeb ? 'svi' : __( 'Small viewport width or height (svi)' ),
```

Comment:
This and `svb` have the same description in their label which confused me since I'm not all that familiar with these new viewport units. It looks like `i` and `b` mean "inline direction" and "block direction", respectively, so I think it would be useful to include those terms wherever `i` and `b` are used.

Maybe something like "Viewport smallest possible size in the inline direction (svi)"? Maybe that's too long? Something like "Viewport smallest inline size (svi)" is shorter, but maybe not very clear for svb, "Viewport smallest block size (svb)". Having such a different meaning for 'block' in Gutenberg makes it tough to be concise here.

Reply:
Thank you for the feedback @ajlende!
I updated the labels 👍